### PR TITLE
add imperative picture loader

### PR DESCRIPTION
**Pull request by jonahwilliams**
Allows customer:money to load a picture so they can draw it into a canvas themselves

### CODE REVIEW

File: packages/vector_graphics/lib/vector_graphics.dart [hunk 1]
```diff
@@ -140,6 +142,25 @@ class VectorGraphic extends StatefulWidget {
   State<VectorGraphic> createState() => _VectorGraphicWidgetState();
 }
 
+/// Load the [PictureInfo] from a given [loader].
+///
+/// It is the caller's responsibility to handle disposing the picture when
+/// they are done with it.
+Future<PictureInfo> loadPicture(
```
Comment (dnfield) [inline packages/vector_graphics/lib/vector_graphics.dart:149]: Should we call this something else?  It's a bare top level method, it seems a little ambiguous.

We could make a vg utility class or something

Comment (jonahwilliams) [reply packages/vector_graphics/lib/vector_graphics.dart:149]: I could add a new method to the loader interface instead perhaps?

Comment (jonahwilliams) [reply packages/vector_graphics/lib/vector_graphics.dart:149]: or call it `LoadPictureForCustomerMoney`

Comment (dnfield) [reply packages/vector_graphics/lib/vector_graphics.dart:149]: In flutter_svg there's a class called `Svg` with a private ctor and `public final Svg svg = Svg._()` for utility methods like this.

That way they don't polute the namespace of whatever imports them and you don't need weird imports like `import 'package:vector_graphics/vector_graphics.dart as vg` and `import package:vector_graphics/vector_graphics.dart show VectorGraphic` or whatever

Comment (dnfield) [reply packages/vector_graphics/lib/vector_graphics.dart:149]: impl looks fine to me though

Comment (jonahwilliams) [reply packages/vector_graphics/lib/vector_graphics.dart:149]: https://dart.dev/guides/language/effective-dart/design#avoid-defining-a-class-that-contains-only-static-members

Comment (dnfield) [reply packages/vector_graphics/lib/vector_graphics.dart:149]: They're not static members.

```dart
class VectorGraphicUtilities {
  VectorGraphicUtilities._();

  Future<Picture> loadPicture(...) {}
}

final VectorGraphicUtilities vg = VectorGraphicUtilities._();
```

Comment (jonahwilliams) [reply packages/vector_graphics/lib/vector_graphics.dart:149]: Technically not against the rules lol.

Comment (jonahwilliams) [reply packages/vector_graphics/lib/vector_graphics.dart:149]: will do

Comment (jonahwilliams) [reply packages/vector_graphics/lib/vector_graphics.dart]: done

**Review by dnfield — APPROVED** [review]

**Pull request merged**